pyproject.toml

```diff
@@ -20,6 +20,9 @@ python-multipart = "0.0.5"
 docxtpl = "0.16.0"
 
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Why? It's not required to run the project, just for convenience during local development. Otherwise docker container wouldn't run successfully without it.